src/Spec.ts

```diff
@@ -740,36 +740,46 @@ export default class Spec {
     });
   }
 
-  private checkValidSectionId(ele: Element) {
+  private readSectionId(ele: Element) {
+    if (ele.id == null) {
+      this.warn({
+        type: 'node',
+        ruleId: 'top-level-section-id',
+        message: 'When using --multipage, top-level sections must have ids',
+        node: ele,
+      });
+      return undefined;
+    }
     if (!ele.id.startsWith('sec-')) {
       this.warn({
         type: 'node',
         ruleId: 'top-level-section-id',
         message: 'When using --multipage, top-level sections must have ids beginning with `sec-`',
         node: ele,
       });
-      return false;
+      return undefined;
     }
-    if (!/^[A-Za-z0-9-_]+$/.test(ele.id)) {
+    const name = ele.id.substring(4);
+    if (!/^[A-Za-z0-9-_]+$/.test(name)) {
```

Comment:
This is technically a behavioral change, but we'd never consider `id="sec-"` to be valid anyway.